realm/kotlin-extensions/build.gradle

```diff
@@ -70,6 +70,7 @@ android {
 }
 
 dependencies {
+    compileOnly 'org.mongodb:bson:3.12.0'
```

Comment:
why adding this, `realm-library` already includes `implementation 'org.mongodb:bson:3.12.0'` ?